docker-compose.yml

```
@@ -46,27 +35,58 @@ services:
+      - TRACK_PROCESSOR_TCP_HOST=track-processor
+      - TRACK_PROCESSOR_TCP_PORT=3001
+    ports:
+      - "3000:3000"
+
+  track-processor:
+    build:
+      context: ./phanto-track-processor
+    container_name: phanto-track-processor
+    restart: always
+    depends_on:
+        mongo:
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)